enhance-merge-requests.js

```diff
@@ -1,5 +1,5 @@
 if (
-  /^https:\/\/gitlab\.com\/.*\/merge_requests\/?(\?.*)?$/.test(
+  /^https:\/\/gitlab\.*.*\/.*\/merge_requests\/?(\?.*)?$/.test(
```

Comment:
Then I think this could be simplified to

```suggestion
  /^https:\/\/gitlab\.*.*\/merge_requests\/?(\?.*)?$/.test(
```

don't you think?